packages/composite-checkout/src/lib/stripe.js

```diff
@@ -242,7 +241,7 @@ function useStripeConfiguration( fetchStripeConfiguration ) {
 			debug( 'loading stripe configuration' );
 			fetchStripeConfiguration()
```

Comment:
Should we be passing in a `wpcom` object here?

Reply:
This should be fixed 🤞